src/registrar/tests/common.py

```diff
@@ -804,7 +805,20 @@ def dummyInfoContactResultData(
     infoDomainHasIP = fakedEppObject(
         "nameserverwithip.gov",
         cr_date=datetime.datetime(2023, 5, 25, 19, 45, 35),
-        contacts=[],
+        contacts=[
+            common.DomainContact(
```

Comment:
why did you add this?

Reply:
For the tests it kept erroring out saying we needed Contacts, with this error `raise ContactError("No contact was found in cache or the registry")`

Rachid, Erin and I thought this was odd too...

Reply:
That is annoying....